src/parser.rs

```diff
@@ -41,34 +47,74 @@ fn directive<'a>(input: &mut &'a str) -> Result<Directive<'a>> {
 }
 
 fn comment<'a>(input: &mut &'a str) -> Result<Directive<'a>> {
-    let res = preceded(
-        (opt(space0), "//", opt(space0)),
-        take_till(0.., AsChar::is_newline),
-    )
-    .parse_next(input)?;
-    let _ = take_while(1.., AsChar::is_newline).parse_next(input)?;
+    let res =
+        preceded((opt(space0), "//", opt(space0)), take_till(0.., NEWLINE)).parse_next(input)?;
+    let _ = take_while(1.., NEWLINE).parse_next(input)?;
 
     Ok(Directive::Comment(res))
 }
 
 fn module<'a>(input: &mut &'a str) -> Result<Directive<'a>> {
-    let res = preceded(("module", space1), take_till(1.., AsChar::is_newline)).parse_next(input)?;
-    let _ = take_while(1.., AsChar::is_newline).parse_next(input)?;
+    let res = preceded(("module", space1), take_till(1.., NEWLINE)).parse_next(input)?;
+    let _ = take_while(1.., NEWLINE).parse_next(input)?;
 
     Ok(Directive::Module(res))
 }
 
 fn go<'a>(input: &mut &'a str) -> Result<Directive<'a>> {
-    let res = preceded(("go", space1), take_till(1.., AsChar::is_newline)).parse_next(input)?;
-    let _ = take_while(1.., AsChar::is_newline).parse_next(input)?;
+    let res = preceded(("go", space1), take_till(1.., NEWLINE)).parse_next(input)?;
+    let _ = take_while(1.., NEWLINE).parse_next(input)?;
 
     Ok(Directive::Go(res))
 }
 
+fn godebug<'a>(input: &mut &'a str) -> Result<Directive<'a>> {
+    let res = preceded(
+        ("godebug", space1),
+        dispatch! {peek(any);
+            '(' => godebug_multi,
+            _ => godebug_single,
+        },
+    )
+    .parse_next(input)?;
+    let _ = take_while(0.., NEWLINE).parse_next(input)?;
+
+    Ok(Directive::GoDebug(HashMap::from_iter(res)))
+}
+
+fn godebug_single(input: &mut &str) -> Result<Vec<(String, String)>> {
+    // terminate, if `)` is found
+    peek(not(')')).parse_next(input)?;
+
+    let (key, _, value) = (
+        take_till(1.., '='),
+        take_while(1.., '='),
```

Comment:
I think this can be simplified to just
```suggestion
        '=',
```